### DISCUSSION TITLE
a VST Store inclusion to Musescore (same way as StaffPad)

### DISCUSSION THREAD

**Discussion by TobyTwo32** [musescore/MuseScore#22970] [General]
### Your idea

The idea would be to introduce good sampled libraries from various retailers and developers like VSL, Orchestral Tools, Spitfire, etc, the same way it is designed in StaffPad. I think this inclusion would approach MuseScore and StaffPad by design. MuseScore would still be free but the samples bought separately 

### Problem to be solved

It would give professional tier playback beyond comparison of what MuseSounds can offer at the moment, and quality playback.

**Comment by Dima-S-Jr** [comment]
Although your suggestion may seem relevant, but personally I would prefer to see MuseScore and its components, including Muse Sounds, which are _free_. And StaffPad is a paid software. Thus (in my opinion) the Muse Sounds team may stop paying attention to their brainchild if it becomes possible _to buy_ paid VST. Although I sincerely hope that this will not happen, both MuseScore Studio and Muse Sounds will continue to be free.

**Comment by Valdessir** [comment]
I beg to differ. As I interpret it, your concern is that this request will result in turning MS studio or Musesounds into a paid product, or maybe you meant including vendor libraries will result in neglecting the development of Musesounds. I don't think that will be the case. What this request likely wants to happen is have the Staffpad libraries, whose samples are provided by VST vendors, that are already available for Staffpad users to also be available for Musescore Studio users to buy and use for Musescore Studio. In short, integrate the premium Staffpad libraries to Musescore Studio. 

Musescore Studio will remain free but add-ons are available for users to purchase. I understand the other likely concern is that Musesounds development will slowly die down. However, I think that'll be unlikely because if the freeware (Musesounds) is buggy, it can leave quite an impression that'll risk sales of premium libraries should they become available.

**Comment by Valdessir** [comment]
Another benefit on Staffpad's side, should they plan on integrating their vendor libraries for Musescore Studio users to purchase and use, is that it will boost sales and boost development--since the aforementioned customer base is the most vocal when it comes to testing and providing feedback.

**Comment by Dima-S-Jr** [comment]
> or maybe you meant including vendor libraries will result in neglecting the development of Musesounds.

> I understand the other likely concern is that Musesounds development will slowly die down.

These two aforementioned things bother me. If they implement the purchase and use of advanced third-party VST plugins in MuseScore Studio (by the way, we should also remember about Note Performer at the same time), then they may stop improving the existing Muse Sounds. Bugs and flaws (including those described by @dcorson-ticino-com in the now-closed issue) will accumulate, and the Muse team will not pay much attention to them. Indeed, why, if there are paid advanced VST.

**Reply by KJinx101** [reply]
I get the perspective, but at the same time, it would be nice if they at the very least could do more about MIDI. Right now it's impossible to change the playback/audio without changing the engraving/score itself... which could address half of the issues we get with musesounds via plugins that would have access to MIDI info.

I don't think they should necessarily focus on paid VSTs, but at least make them more operable. (And even make musesounds a VST or something as well(?))
